packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -60,6 +60,7 @@ import { useContentPreference } from '../hooks/contentPreference/useContentPrefe
 import { ContentPreferenceType } from '../graphql/contentPreference';
 import { isFollowingContent } from '../hooks/contentPreference/types';
 import { useIsSpecialUser } from '../hooks/auth/useIsSpecialUser';
+import { FolderIcon } from './icons/Folder';
```

Comment:
This should import from icons directly ideally as there's more then 1 import from icons here.